Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.h

```diff
@@ -343,6 +347,12 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsNode : public vtkMRML
   /// scene. Returns false if n out of bounds, true on success.
   bool ResetNthMarkupID(int n);
 
+  /// Utility function to convert a markup point position in world coordinate system
```

Comment:
Unify comment strings between TransformPointToWorld and TransfromPointFromWorld.